apiary.apib

```diff
@@ -3,128 +3,156 @@ HOST: http://lore.mit.edu/
 
-Page numbering is 1-based. Omitting the ?page parameter will return the first page.
+Page numbering is 1-based. Omitting the ?page parameter will return the first page. There are up to 20 results per page.
 
-# LORE API Root [/]
+# LORE API
 
-## Retrieve the Entry Point [GET]
+## Root [/]
 
-+ Response 200 (application/json)
+### Retrieve the Entry Point [GET]
+
++ Response 200 (application/JSON)
 
         {
-            "repositories_url": "/repos"
+            "repositories": "http://lore.mit.edu/api/v1/repositories/"
         }
 
 
 
 ## Group Repositories
 
-Repository resource
-
-## Repositories [/repos/{repo_name}]
-
-+ Parameters
-    + repo_name: `physics-intro` (string, required) - name of the repository
+## Repository Collection [/repositories/]
 
-### Retrieve a repository [GET]
+### List All Repositories [GET]
 
-+ Response 200 (application/json)
++ Response 200 (application/JSON)
 
-        {
-            "tbd": "tbd",
-        }
+        [
+            "count": 1,
+            "next": null,
+            "previous": null,
+            "results": [
+                {
+                    "id": 1,
+                    "slug": "repository-name",
+                    "name": "repository name",
+                    "description": "repository description",
+                    "create_date": "2015-06-11"
+                }
+            ]
+        ]
 
-### Delete a repository [DELETE]
-Delete the repository and all its associated resources (learning resources, vocabularies etc.)
-
-+ Response 204
 
 ### Create a New Repository [POST]
 
 Create a repository by providing its JSON representation
 
-+ Request (application/json)
++ Request (application/JSON)
 
         {
             "name": "Physics 1",
             "description": "All intro-to-physics courses"
         }
 
-+ Response 201 (application/json)
++ Response 201 (application/JSON)
 
     + Headers
 
-            Location: /repos/physics-intro
+            Location: /api/v1/repositories/physics-1/
 
     + Body
 
             {
-                "name": "physics-intro",
-                "...": "..."
-                "tbd": "Full json representation of the new repo goes here
+                "id": 8,
+                "slug": "physics-1",
+                "name": "Physics 1",
+                "description": "All intro-to-physics courses",
+                "create_date": "2015-06-17"
             }
 
+## Repository [/repositories/{repo_slug}/]
+
++ Parameters
+    + repo_slug: `physics-intro` (string, required) - slug for the repository
+
+### Retrieve a Repository [GET]
+
++ Response 200 (application/JSON)
+
+        {
+            "id": 1,
+            "slug": "repository-name",
+            "name": "repository name",
+            "description": "repository description",
+            "create_date": "2015-06-11"
+        }
+
+### Delete a Repository [DELETE]
+Delete the repository and all its associated resources (learning resources, vocabularies etc.)
```

Comment:
This sentence needs a final period.  The period in "etc." is part of its spelling and isn't a full stop for the sentence.